index.html

```diff
@@ -1,9 +1,9 @@
 <html>
 <head>
```

Comment:
Can you add the meta charset so Spanish users can write in their language? They want to use Spanish accents

<meta charset="UTF-8" />